aws_iam_account_password_policy.tf

```diff
@@ -4,40 +4,40 @@
 
 
 resource "aws_iam_account_password_policy" "passwordRequireMinLength" {
-  minimum_password_length        = 6
+  minimum_password_length = 14
 }
 
 resource "aws_iam_account_password_policy" "passwordRequireMinLength14" {
-  minimum_password_length        = 13
+  minimum_password_length = 14
 }
 
 resource "aws_iam_account_password_policy" "passwordRequireLowerCase" {
-  require_lowercase_characters   = false
-  minimum_password_length        = 14
+  require_lowercase_characters = true
+  minimum_password_length      = 14
 }
 
 resource "aws_iam_account_password_policy" "passwordRequireNumber" {
-  require_numbers                = false
-  minimum_password_length        = 14
+  require_numbers         = true
+  minimum_password_length = 14
```

Comment:
**Accurics remediation**
 
**[MEDIUM]: Password Policy must require at least one uppercase character**
 
**Vulnerability**
Upper case alphabet not present in the Password, Password Complexity is not high. Increased Password complexity increases resiliency against brute force attack
 
**Remediation**
Require atleast one upper case alphabet in Password to increase Complexity

source: accurics-